src/Common/ExtraType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Common;
```

Comment:
Indeed, Moving everything to Common is the same like creating another spoon library.